lib/mongoid/fields/localized.rb

```diff
@@ -29,6 +29,16 @@ def localized?
         true
       end
 
+      # Is the localized field enforcing values to be present?
+      #
+      # @example Is the field localized?
+      #   field.present?
+      #
+      # @return [ true | false ] If the field enforces present.
+      def present?
```

Comment:
`present?` already is defined on Object by ActiveSupport and should be treated as a reserved method name. I propose to rename this to `remove_blank_localized_values?` or something similarly specific, since its also defined on Fields::Standard.